Gemfile

```diff
@@ -31,30 +31,30 @@ else
 end
 
 gem 'govspeak', '~> 3.1'
-gem 'plek', '1.7.0'
+gem 'plek', '1.10.0'
 gem 'yajl-ruby'
 gem 'kaminari', '0.14.1'
-gem 'link_header', '0.0.5'
+gem 'link_header', '0.0.8'
 gem 'rack-cache', '1.2'
-gem 'dalli', '2.6.4'
+gem 'dalli', '2.7.4'
 
 gem 'rack-logstasher', '0.0.3'
-gem 'airbrake', '4.0.0'
+gem 'airbrake', '4.3.0'
 
 group :test do
-  gem 'database_cleaner', '0.7.2'
-  gem 'factory_girl', '3.6.1'
+  gem 'database_cleaner', '1.4.1'
+  gem 'factory_girl', '4.5.0'
   gem 'mocha', '0.12.4', require: false
-  gem 'simplecov', '0.6.4'
+  gem 'simplecov', '0.10.0'
   gem 'simplecov-rcov', '0.2.3'
   gem 'minitest', '3.4.0'
   gem 'turn', require: false
-  gem 'ci_reporter', '1.7.0'
-  gem 'webmock', '~> 1.8', require: false
-  gem 'timecop', '0.5.9.2'
+  gem 'ci_reporter', '2.0.0'
```

Comment:
See https://github.com/ci-reporter/ci_reporter#upgrading-from-cireporter-1x for details.